drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -377,6 +377,15 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 					/ (CLOCK_GetDiv(kCLOCK_Flexio2Div) + 1);
 	} break;
 #endif
+
+#ifdef CONFIG_ADC_MCUX_LPADC
+	case IMX_CCM_LPADC1_CLK:
```

Comment:
Is there a part using the CCM driver that has an LPADC?

Reply:
Hi Daniel, the I.MX RT 11xx series use the CCM